modules/convert.py

```diff
@@ -71,7 +71,7 @@ def unit_select(unit, level):
     units = sheets.get_col('unit')
     # Level 3, Crazy Units Only
     if(sheets.level == 3):
-        units = units[9:]
+        units = units[10:]
```

Comment:
why is it changed to 10?

Reply:
I added meters to the sheets document for SI -> SI conversion testing, so the obscure units start at row 10 now instead of row 9